src/javascript/common/champion.js

```diff
@@ -94,7 +94,9 @@ const Champion = (function() {
             cfd                    : { module: MT5 },
             metals                 : { module: MT5 },
             profile                : { module: ChampionProfile,     is_authenticated: true },
-            real                   : { module: ChampionNewReal,     is_authenticated: true, only_virtual: true },
+            /* eslint-disable */
+            real                   : { module: ChampionNewReal,     is_authenticated: true, only_virtual: true, is_disabled: true },
```

Comment:
you can just put `// eslint-disable-line [name of rule to disable]` at the end of line 98